dll/win32/shell32/shlexec.cpp

```diff
@@ -32,6 +32,32 @@ EXTERN_C BOOL PathIsExeW(LPCWSTR lpszPath);
 typedef UINT_PTR (*SHELL_ExecuteW32)(const WCHAR *lpCmd, WCHAR *env, BOOL shWait,
                 const SHELLEXECUTEINFOW *sei, LPSHELLEXECUTEINFOW sei_out);
 
+class CHeapAllocator
+{
+public:
+    static void* Allocate(_In_ size_t size)
+    {
+        return ::HeapAlloc(::GetProcessHeap(), 0, size);
```

Comment:
Any reason why this should be the process heap, and you can't use `CCRTAllocator` ?

Reply:
Used `CLocalAllocator` instead.